### ISSUE TITLE
Could get used memory on my device, not on app

### ISSUE THREAD

**Issue by trinhcanhphuc** [react-native-device-info/react-native-device-info#1585]
## Summary

|             |     |
| ----------- | --- |
| Version     | ?   |
| Affected OS | ?   |
| OS Version  | ?   |

## Current behavior

I am using function getUsedMemory() to get used memory of android system, but it show memory of this app. How do I get used memory on device system? Thanks

**Comment by trinhcanhphuc** [comment]
Could you add method get available memory?

**Comment by mikehardy** [comment]
No. I interpret your request, after the detail I posted in my comment, as if you did not read my comment.
I will not interact with this issue anymore.
If there is some API that does what you want, and still works on modern Android versions, you are welcome to post a PR.
Reasonable PRs are merged here.

**Comment by trinhcanhphuc** [comment]
Thanks for your explanation. I got it
